localstack/config.py

```diff
@@ -579,6 +579,12 @@ def in_docker():
     """
     if OVERRIDE_IN_DOCKER:
         return True
+
+    # check things from Dockerfile
+    # TODO: is it enough? can we remove all the checks bellow?
+    if __file__.startswith("/opt/code/localstack/") and os.environ.get("USER") == "localstack":
+        return True
```

Comment:
as we discussed, maybe it would be better to
- [ ] set OVERRIDE_IN_DOCKER=1 in the Dockerfile
- [ ] replace the if statement with `os.path.exists("/run/.containerenv")` as @dfangl suggested

Reply:
changed to `os.path.exists("/run/.containerenv")`